README.md

```diff
@@ -31,37 +29,37 @@ npm install react-countdown360
 Render your countdown:
 
 ```js
-import Countdown360 from 'react-countdown360'
+import Countdown360 from "react-countdown360";
 
 const App = () => {
-  return <Countdown360 seconds={10} />
-}
+  return <Countdown360 seconds={10} />;
+};
 ```
 
 ## Documentation
 
 ### Props
 
-| Name                | Type     | Default                 | Description                                     |
-|---------------------|----------|-------------------------|-------------------------------------------------|
-| `seconds`           | Number   | -                       | Number of seconds for the countdown             |
-| `autoStart`         | Boolean  | `true`                  | Start the countdown immediatly after rendering  |
-| `backgroundColor`   | String   | `'#fff'`                | Color for the center of the circle              |
-| `borderFillColor`   | String   | `'#f11'`                | Color for the filled part of the border         |
-| `borderUnfillColor` | String   | `'#e6e2e7'`             | Color for the unfilled part of the border       |
-| `borderWidth`       | Number   | 20                      | Width in pixels of the border                   |
-| `clockwise`         | Boolean  | `false`                 | Select the direction of rotation you prefer     |
-| `fontColor`         | String   | `'#111'`                | Font color for the label                        |
-| `fontFamily`        | String   | `'sans-serif'`          | The font to use for the label                   |
-| `fontSize`          | Number   | 45                      | Font size in pixels                             |
-| `fontWeight`        | Number   | 700                     | Font weight for the label                       |
-| `onComplete`        | Function | `undefined`             | A callback called when the countdown is over    |
-| `smooth`            | Boolean  | `false`                 | Update the border once every second or smoothly |
-| `startingAngle`     | Number   | 0                       | The angle at which the countdown should start   |
-| `timeFormatter`     | Func     | `timeFormatterSeconds`  | A function that returns the value to display    |
-| `unitFormatter`     | Func     | `unitFormatterSeconds`  | A function that returns the unit to display     |
-| `width`             | Number   | 200                     | Width in pixels of the countdown to render      |
-
+| Name                | Type     | Default                | Description                                                    |
+| ------------------- | -------- | ---------------------- | -------------------------------------------------------------- |
+| `seconds`           | Number   | -                      | Number of seconds for the countdown                            |
+| `remainingSeconds`  | Number   | -                      | Number of remaining seconds from total seconds (seconds props) |
```

Comment:
The default value is actually `undefined`. I use the `-` notation to refer to mandatory parameters.

Reply:
Maybe a better naming would be something like `startAt` (or other similar idea)? Because `props.remainingSeconds` is actually a constant and is not updated when the timer is on.

Reply:
Also, sort the new parameter by alphetical order. The only exception is for `seconds`, which is the only mandatory parameter and should stay on top.

Reply:
> Maybe a better naming would be something like `startAt` (or other similar idea)? Because `props.remainingSeconds` is actually a constant and is not updated when the timer is on.

I agree, how about startingSecond? i think it will also be consistent with startingAngle

Reply:
Yup, `startingSecond` sounds good